admin/test/test_release.py

```diff
@@ -697,7 +694,7 @@ def test_production_can_publish_prerelease(self):
             })
         # Does not raise:
         self.publish_docs(
-            aws, '0.3.2pre1', '0.3.2pre1', environment=Environments.PRODUCTION)
+            aws, '0.3.2rc1', '0.3.2rc1', environment=Environments.PRODUCTION)
```

Comment:
The remaining tests containing `pre` check for correct behaviour with legacy version numbers.